source/extensions/filters/http/ext_authz/ext_authz.h

```diff
@@ -275,8 +285,10 @@ class Filter : public Logger::Loggable<Logger::Id::filter>,
   Http::HeaderMapPtr getHeaderMap(const Filters::Common::ExtAuthz::ResponsePtr& response);
   FilterConfigSharedPtr config_;
   Filters::Common::ExtAuthz::ClientPtr client_;
-  Http::StreamDecoderFilterCallbacks* callbacks_{};
+  Http::StreamDecoderFilterCallbacks* decoder_callbacks_{};
+  Http::StreamEncoderFilterCallbacks* encoder_callbacks_{};
   Http::RequestHeaderMap* request_headers_;
+  std::unique_ptr<Http::HeaderVector> response_headers_to_add_{};
```

Comment:
Nit. I think this should be type aliased. ref: STYLE.md

Reply:
Good callout, but I'll try updating this to a non-pointer, actually.